vega_sim/scenario/ideal_market_maker_v2/scenario.py

```diff
@@ -208,12 +210,34 @@ def set_up_background_market(
             tag=str(tag),
         )
 
+        momentum_trader = MomentumTrader(
+            wallet_name=MOMENTUM_WALLET.name,
+            wallet_pass=MOMENTUM_WALLET.passphrase,
+            market_name=market_name,
+            asset_name=asset_name,
+            initial_asset_mint=self.initial_asset_mint,
+            buy_intensity=self.buy_intensity,
+            sell_intensity=self.sell_intensity,
+            base_order_size=self.market_order_trader_base_order_size,
+            indicator_threshold=(70, 30),
+            momentum_strategy=(
+                "RSI",
+                {
+                    "period": 14,
+                },
+            ),
+            send_limit_order=True,
+            offset_levels=20,
+            tag=str(tag),
+        )
```

Comment:
I wouldn't necessarily add this to the ideal market maker v2 since that's a fairly 'pure' one but maybe let's add to the comprehensive scenario?